test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,45 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() {
+
+        // some null
```

Comment:
Maybe describe the purpose of the test cases instead of just the expected output?